src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/index.js

```diff
@@ -54,7 +54,7 @@ const generateAnnotationData = (values, formatter) =>
 const decorateFormatter = (formatter) => ({ value }) => formatter(value);
 
 const handleCursorUpdate = (cursor) => {
-  eventBus.trigger(ACTIVE_CURSOR, cursor);
+  eventBus.next({ eventName: ACTIVE_CURSOR, cursor });
```

Comment:
are you sure we need to pass `eventName: ACTIVE_CURSOR`

Reply:
removed it